### PR TITLE
Readme writeup

### PR DESCRIPTION
**Pull request by Kabir404**
This would clear things out for everyone. Initially written by myself but rewritten again by OpenAI ChatGPT

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,30 +3,31 @@ SPDX-FileCopyrightText: 2023 Kabir Akzaman <kaabirherococ@gmail.com>
...
-#### What does it contain?
-It contains files such as...
+The repository includes the following files:
 
-+ Game's story
-+ "Screenplay" script
-+ Logos and other core branding
-+ Planning for development
++ **Game Story** - Provides the backstory and plotline for Depth Walker
++ **Screenplay Script** - Includes the screenplay for the game
```
Comment (mdhishamsayed) [inline README.md:15]: remove word script

File: README.md [hunk 2]
```diff
@@ -3,30 +3,31 @@ SPDX-FileCopyrightText: 2023 Kabir Akzaman <kaabirherococ@gmail.com>
...
 
-Within Kioxia, the notorious criminal organization known as DarkWave has been engaging in illegal activities, supplying cutting-edge technology to criminals. When DarkWave's human personnel evaded capture, they left behind sensitive information guarded by their dangerous machinery.
+In the game, players take on the role of RektifierKX, an advanced robot agent created by Mechmorphix, a reputable AI technology company working with law enforcement agencies in Kioxia city.
 
-As RektifierKX, players must embark on a stealth-based infiltration mission into the abandoned DarkWave facility. Using their exceptional skills, players must gather crucial intel while avoiding detection, unraveling the criminal organization's secrets.
+The notorious DarkWave criminal organization has been conducting illegal activities within Kioxia, providing high-tech gadgets to criminals. However, when the organization's human personnel escaped, they left behind valuable information under the protection of their dangerous machinery.
```
Comment (mdhishamsayed) [inline README.md:25]: add some introduction to the darkwave

File: README.md [hunk 3]
```diff
@@ -3,30 +3,31 @@ SPDX-FileCopyrightText: 2023 Kabir Akzaman <kaabirherococ@gmail.com>
...
 
-Through intense action, strategic decision-making, and immersive stealth gameplay, players will explore the depths of the facility, uncovering a web of corruption and criminal activities that must be stopped.
+As RektifierKX, players must infiltrate the abandoned DarkWave facility stealthily, collect information and avoid detection. By using exceptional skills and making strategic decisions, players uncover secrets and stop criminal activities to restore order to Kioxia.
```
Comment (mdhishamsayed) [inline README.md:27]: law and order

File: README.md [hunk 4]
```diff
@@ -3,30 +3,31 @@ SPDX-FileCopyrightText: 2023 Kabir Akzaman <kaabirherococ@gmail.com>
...
+The repository includes the following files:
 
-+ Game's story
-+ "Screenplay" script
-+ Logos and other core branding
-+ Planning for development
++ **Game Story** - Provides the backstory and plotline for Depth Walker
++ **Screenplay Script** - Includes the screenplay for the game
++ **Logos and Core Branding** - Contains essential branding elements
++ **Development Planning** - Includes necessary files crucial to the development process
```
Comment (mdhishamsayed) [inline README.md:17]: steps, like first develop backend then graphics etc

File: README.md [hunk 5]
```diff
@@ -3,30 +3,31 @@ SPDX-FileCopyrightText: 2023 Kabir Akzaman <kaabirherococ@gmail.com>
...
 
-## What is even "Depth Walker"?
-Depth Walker, directed by Kabir Akzaman and developed by the OpenNullanoid community contributors, is an exhilarating Sci-Fi Action-Adventure First Person Shooter (FPS) project.
+## What is Depth Walker?
 
-In this thrilling game, players assume the role of RektifierKX, an advanced robot agent developed by Mechmorphix, an esteemed AI technology company working in partnership with the Law Enforcement agencies of Kioxia city.
+Depth Walker is an action-packed first-person shooter (FPS) game that falls under the sci-fi action-adventure genre. Directed by Kabir Akzaman and developed by OpenNullanoid community contributors, Depth Walker is an exciting project to work on.
```
Comment (mdhishamsayed) [inline README.md:21]: exciting game to play

File: README.md [hunk 6]
```diff
@@ -3,30 +3,31 @@ SPDX-FileCopyrightText: 2023 Kabir Akzaman <kaabirherococ@gmail.com>
...
 
-Within Kioxia, the notorious criminal organization known as DarkWave has been engaging in illegal activities, supplying cutting-edge technology to criminals. When DarkWave's human personnel evaded capture, they left behind sensitive information guarded by their dangerous machinery.
+In the game, players take on the role of RektifierKX, an advanced robot agent created by Mechmorphix, a reputable AI technology company working with law enforcement agencies in Kioxia city.
```
Comment (mdhishamsayed) [inline README.md:23]: use some simpler names

**Review by mdhishamsayed — COMMENTED** [review]
Kabir is requested to look into the necessary changes

**Review by mdhishamsayed — CHANGES_REQUESTED** [review]
Kabir is requested to look into the necessary changes

**Review by mdhishamsayed — CHANGES_REQUESTED** [review]
Kabir is requested to look into the necessary changes

**Comment by Kabir404** [comment]
> Kabir is requested to look into the necessary changes

I would look into that later

File: README.md (hunk 6 above)
Comment (Kabir404) [reply README.md:23]: RektifierKX has its nickname which is Rex while DarkWave and Mechamorphix are still reasonable...

File: README.md (hunk 4 above)
Comment (Kabir404) [reply README.md:17]: Yes, yes, that would be in the documentation hosted in this repo...

File: README.md (hunk 1 above)
Comment (Kabir404) [reply README.md:15]: I dont think thats that big of an issue...

File: README.md (hunk 2 above)
Comment (Kabir404) [reply README.md]: did it

**Review by mdhishamsayed — APPROVED** [review]

**Pull request merged**